app/schemas/mega_sena.py

```diff
@@ -0,0 +1,8 @@
+from app.schemas import ma
+from app.models import mega_sena
```

Comment:
Pode simplificar o import aqui novamente.
```suggestion
from app.models import MegaSena
```